### PR TITLE
DOCS-4146: DateTime. EN review, RU translation for #7429

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/data_types/datetime.md [hunk 1]
```diff
@@ -1,45 +1,41 @@
 # DateTime {#data_type-datetime}
 
-Data structure storing Unix timestamp. Also, it can store a time zone.
+Data structure for storing [Unix time](https://en.wikipedia.org/wiki/Unix_time). It can also store a time zone.
```
Comment (blinkov) [inline docs/en/data_types/datetime.md]: "timestamp" is a proper word here, "time" is some rules how these timestamps are supposed to be treated.
Meanwhile when there's a timezone it stops being Unix time. Needs to be fixed in all following text.

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by blinkov** [comment]
also there's a file conflict

File: docs/en/data_types/datetime.md [hunk 2]
```diff
@@ -1,45 +1,54 @@
...
+INSERT INTO dt Values (1546300800), ('2019-01-01 00:00:00')
+```
+```sql
+SELECT * FROM dt
+```
+```text
+┌───────────timestamp─┐
+│ 2019-01-01 03:00:00 │
+│ 2019-01-01 00:00:00 │
+└─────────────────────┘
```
Comment (den-crane) [inline docs/en/data_types/datetime.md]: I would add some ID column
INSERT INTO dt Values (1, 1546300800), (2, '2019-01-01 00:00:00');

and example how to select it with a condition

select * from dt where timestamp = toDateTime('2019-01-01 00:00:00', 'Europe/Moscow')

Comment (BayoNet) [reply docs/en/data_types/datetime.md]: Thank you a lot, I'll add it.

File: docs/en/data_types/datetime.md [hunk 3]
```diff
@@ -1,45 +1,70 @@
...
+SELECT * FROM dt
+```
+```text
+┌───────────timestamp─┬─event_id─┐
+│ 2019-01-01 03:00:00 │        1 │
+│ 2019-01-01 00:00:00 │        2 │
+└─────────────────────┴──────────┘
+```
+
+Unix timestamp `1546300800` represents the `'2019-01-01 00:00:00'` date and time in `Europe/London (UTC+0)` time zone, but the `timestamp` column stores values in the `Europe/Moscow (UTC+3)` timezone, so the value inserted as Unix timestamp represents the `2019-01-01 03:00:00` date and time.
```
Comment (blinkov) [inline docs/en/data_types/datetime.md]:
```suggestion
Unix timestamp `1546300800` represents the `'2019-01-01 00:00:00'` date and time in `Europe/London` (UTC+0) time zone, but the `timestamp` column stores values in the `Europe/Moscow` (UTC+3) timezone, so the value inserted as Unix timestamp is formatted as `2019-01-01 03:00:00`.
```

**Review by blinkov — CHANGES_REQUESTED** [review]

**Pull request merged**